src/utils/Dom.ts

```diff
@@ -129,7 +129,25 @@ export class Dom {
    */
   public empty(): void {
     while (this.el.firstChild) {
-      this.el.removeChild(this.el.firstChild);
+      this.removeChild(this.el.firstChild);
+    }
+  }
+
+  /**
+   * Safely removes a child, only suppressing `NotFoundError` if the child's parent was changed.
```

Comment:
I would remove this comment, because it does not give information beyond what the code is doing.

Reply:
Good point, removing it